brainrender_napari/widgets/atlas_manager_view.py

```diff
@@ -100,24 +100,28 @@ def update_fn(completed, total):
         worker.start()
 
     def _on_download_atlas_confirmed(self):
-        """Downloads the currently selected atlas and signals this."""
         atlas_name = self.selected_atlas_name()
         self._start_worker(
             install_atlas,
             atlas_name,
             self.download_atlas_confirmed,
             "Downloading",
         )
+        self._refresh_cached_atlases()
 
     def _on_update_atlas_confirmed(self):
-        """Updates the currently selected atlas and signals this."""
         atlas_name = self.selected_atlas_name()
         self._start_worker(
             update_atlas,
             atlas_name,
             self.update_atlas_confirmed,
             "Updating",
         )
+        self._refresh_cached_atlases()
+
+    def _refresh_cached_atlases(self):
+        self.cached_downloaded_atlases = get_downloaded_atlases()
+        self.cached_atlas_versions = get_atlases_lastversions()
```

Comment:
can we not just call `self.model().refresh_data` instead (and cache things in that function instead if we really need to improve performace)?